mb/mb/testmirror.py

```diff
@@ -38,13 +38,19 @@ def req(baseurl, filename, http_method='GET', get_digest=False):
 
 def probe(S, http_method='GET'):
 
-    if S.scheme in ['http', 'ftp']:
+    if S.scheme in ['http', 'https', 'ftp']:
         req = urllib.request.Request(S.probeurl)
         if S.scheme == 'http' and http_method == 'HEAD':
             # not for FTP URLs
             req.get_method = lambda: 'HEAD'
 
         try:
+            if S.scheme == 'https':
+                try:
+                    import ssl
+                    ssl._create_default_https_context = ssl._create_unverified_context
```

Comment:
you dont want that. if cert validation fails already in mb add, it will probably also fail for our users. we should notice this early.

Reply:
Without this self signed certificates will not work. And since we aim to keep py2 behavior, Lars suggested that for now we may need accept self signed certificates. (And cert validation will work properly in all other aspects)

Reply:
This might still make sense to have it as a fallback and warn for the fallback

something like this pseudo code

```
try:
  test_http_connection(host, unverified=false)
except ssl.SSLCertVerificationError:
  print_warning("Potentially broken SSL cert on host %s", host)
  test_http_connection(host, unverified=true)
```

Just siltently falling through is bad IMHO and we will never notice it to notify our mirrors

Reply:
I investigated closer py2 vs p3 behavior and haven't found much difference here, so we should rely on default certificate validation, so this code was removed.